tests/test_linux_support.py

```
@@ -72,11 +72,21 @@ def test_collect_linux_records_reads_bashrc_and_etc_environment(tmp_path: Path):
     rows = collect_linux_records(bashrc_path=bashrc, etc_environment_path=etc_env, context="linux")
 
     source_types = {r.source_type for r in rows}
-    assert "linux_bashrc" in source_types
-    assert "linux_etc_environment" in source_types
-    assert all(r.context == "linux" for r in rows)
-    assert any(r.name == "API_TOKEN" and r.value == "abc123" for r in rows)
-    assert any(r.name == "LANG" and r.value == "en_US.UTF-8" for r in rows)
+    if not ("linux_bashrc" in source_types):
```

Ruff (0.15.2)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)